extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1498,13 +1502,18 @@ export class AmpA4A extends AMP.BaseElement {
    * TODO(keithwrightbos@): report error cases
    */
   getAmpAdMetadata_(creative) {
```

Comment:
@jridgewell I'd like to get rid of all these variants though. Do you think it's worth having a regex in the meantime?